### PR TITLE
View local restaurants back end

### PR DESCRIPTION
**Pull request by jeff-jones2020**
image

### CODE REVIEW

File: server/public/index.html [hunk 1]
```diff
@@ -19,6 +19,7 @@
       crossorigin="anonymous"
     />
     <link rel="stylesheet" href="/styles.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.js"></script>
```
Comment (jinho6225) [inline server/public/index.html]: do we need this?

Comment (jinho6225) [reply server/public/index.html]: there is conflict on index.js
i checked that we need to remain everything.
check together and merge it tomorrow

Comment (jeff-jones2020) [reply server/public/index.html]: No, I forgot to remove this when I was trying different things to get Yelp requests to work. I have pushed another commit that removes this.
Nice catch :)

**Review by jinho6225 — APPROVED** [review]
looks good! good job

**Pull request merged**